sips/instance_decided_enforcement.md

```diff
@@ -0,0 +1,150 @@
+| Author      | Title                        | Category | Status |
+|-------------|------------------------------|----------|--------|
+| Alon Muroch | Instance Decided Enforcement | Core     | open-for-discussion  |
+
+**Summary**
+
+No longer require a QBFT instance to decide in order to start the next one.
+If an instance is stalled, it will be terminated after a certain round or when a new instance starts.
+The instance's height will be the duty's slot number. This will allow for easier syncing.
```

Comment:
I see some confusion here.  In our logs we will have logs concerning height and some concerning slot.  I feel users will ask what is the difference here as it was my first question.  I understand we want to use "height" instead of slot to stay with proper QBFT terminology but maybe its worth removing "slot" terminology as well.